tests/core/request.test.ts

```diff
@@ -237,6 +237,30 @@ describe('Request properties', () => {
 
       await fetch('/').expect(200, 'subdomains: ')
     })
+    it('assigns req.hostname', async () => {
```

Comment:
Change the test name.

Reply:
To ...? have you looked at the test names in context

Reply:
`should assign hostname to req.hostname`

Reply:
@Lordfirespeed I hope the test name is pellucid. It tells you what test is being performed.